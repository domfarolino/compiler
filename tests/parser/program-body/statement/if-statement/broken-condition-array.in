program dominic is

  integer array[0: 9];
  integer i;

begin

  if (array) then
    i := 1;
  end if;

end program.
