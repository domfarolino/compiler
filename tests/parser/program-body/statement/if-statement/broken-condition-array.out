[1;31mLine 8: If statement must be a boolean or boolean equivalent, not integer array
Line 8: Error parsing program body
[0m